src/deepqmc/wf/paulinet/paulinet.py

```diff
@@ -301,7 +272,14 @@ def from_pyscf(
             mf.mol.spin,
         )
         basis = GTOBasis.from_pyscf(mf.mol)
-        wf = cls(mol, basis, **{'omni_factory': OmniSchNet, **kwargs})
+        wf = cls(
```

Comment:
This change isn't needed, you need to remove the trailing comma, then black will collapse it.